test/test_helper.rb

```diff
@@ -4,6 +4,7 @@
 require 'minitest/autorun'
 require 'mocha/setup'
 require 'timecop'
+require 'byebug'
```

Comment:
Do we really need it here?

Reply:
I needed it for debugging during testing. Don't see a reason to not include it.